build.sbt

```diff
@@ -10,9 +10,6 @@ spName := "databricks/spark-xml"
 
 crossScalaVersions := Seq("2.11.12", "2.12.8")
 
-// Necessary for JUnit tests to be found?
-crossPaths := false
```

Comment:
I don't think it was true .. this was ported from Spark CSV before.

I see Java tests are being ran fine:

```
[info] Test com.databricks.spark.xml.JavaXmlSuite.testLoad started
[info] Test com.databricks.spark.xml.JavaXmlSuite.testSave started
[info] Test com.databricks.spark.xml.JavaXmlSuite.testXmlParser started
```